browser/about_flags.cc

```diff
@@ -121,7 +121,7 @@ const flags_ui::FeatureEntry::FeatureVariation kZCashFeatureVariations[] = {
           "brave-speedreader",                                             \
           "Enable SpeedReader",                                            \
           "Enables faster loading of simplified article-style web pages.", \
-          kOsDesktop | kOsAndroid,                                         \
```

Comment:
We should remove it from a few more places:
- features exposed to Java https://github.com/brave/brave-core/blob/master/chromium_src/chrome/browser/flags/android/chrome_feature_list.cc#L52
- Settings UI where we check on that flag https://github.com/brave/brave-core/blob/master/android/java/org/chromium/chrome/browser/settings/AppearancePreferences.java#L83

Reply:
@SergeyZhukovsky I believe the locations mentioned above depend on the `brave-speedreader` feature being enabled. So, if the user isn't able to enable it via brave://flags, it wouldn't appear in the settings.
My understanding is that if we decide to re-enable it in the future, we just need to make the flag available again in brave://flags, and everything should work as expected.
Please let me know if I'm mistaken.

Reply:
I don't think that feature is supported, we just want to remove it. We can always find that PR if we ever decide to bring it back.